cmd/zpool/zpool_main.c

```diff
@@ -676,6 +676,20 @@ zpool_do_add(int argc, char **argv)
 		return (1);
 	}
 
+	/* unless manually specified use "ashift" pool property for new vdevs */
+	if (!nvlist_exists(props, ZPOOL_CONFIG_ASHIFT)) {
```

Comment:
The pool property cannot be changed, so this would be rather annoying. I don't see it serving much purpose as it is now, so we should be able to change it to stop being a onetime thing. Then this would make more sense.

I do not think @ahrens would like this though.